db/config.cc

```diff
@@ -949,7 +949,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "\n"
         "* priority_string: GnuTLS priority string controlling TLS algorithms used/allowed.")
     , ssl_storage_port(this, "ssl_storage_port", value_status::Used, 7001,
-        "The SSL port for encrypted communication. Unused unless enabled in encryption_options.")
+        "The SSL port for encrypted communication. Unused unless enabled in server_encryption_options or client_encryption_options. Set to 0 to prevent server from listening on this port.")
```

Comment:
This is inaccurate and also incomplete since it doesn't cover the storage_port that is also controlled in a similar way by the server_encryption_options.

The client_encryption_options control the client-facing port, not the storage ports, that are used for inter-node communications.  Yes, I know their names suck...

In any case, @elcallio needs to update the documentation as part of https://github.com/scylladb/scylladb/pull/18939, so let's do that there.

Reply:
i see. so the `ssl_storage_port` and `storage_port` are only for intra-node communications, and they are governed by `server_encryption_options`.

i am closing this PR.